### PR TITLE
Moved conjunction_1 below community strings

### PR DESCRIPTION
**Pull request by rolfedh**
The purpose of this PR is to move [conjunction_1] to its correct location.

### CODE REVIEW

**Comment by rolfedh** [comment]
I believe this PR is ready for merge.

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -1,12 +1,7 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
 
-# Avoid using conjunction_1.
-# Must match the context in layouts/shortcodes/release-data.html
-# Appears on https://kubernetes.io/releases/
-# For example the "and" in "Complete 1.25 Schedule and Changelog"
-[conjunction_1]
-other = "and"
+# For "and", see [conjunction_1]
```
Comment (Sea-n) [inline data/i18n/en/en.toml:4]: Maybe it's unnecessary, how do you think?

File: data/i18n/en/en.toml [hunk 2]
```diff
@@ -35,6 +30,12 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
```
Comment (Sea-n) [inline data/i18n/en/en.toml:36]: Not sure if it's good practice to limit the use of a string with general name.

File: data/i18n/en/en.toml (hunk 1 above)
Comment (rolfedh) [reply data/i18n/en/en.toml:4]: I think it is helpful. Users will most likely search alphabetically by the string value, "and", not by the string name, "conjunction_1".

File: data/i18n/en/en.toml (hunk 2 above)
Comment (rolfedh) [reply data/i18n/en/en.toml:36]: The conjunction is useful in a specific context. However, if it is used in many contexts, not all those contexts might localize the same way.

**Pull request merged**